third_party/ReadMe.txt

```diff
@@ -69,4 +69,9 @@ For Windows:
    PATH environment variable. You can use the methods listed in
    https://support.microsoft.com/en-us/help/131313/how-to-create-32-bit-import-libraries-without--objs-or-source.
 6. The steps 3-5 are for 64 bit binaries. For 32 bit binaries, do the same steps but copy the header files and DLLs from
-   <MSYS inst dir>\mingw32\include and <MSYS inst dir>\mingw32\bin respectively.
\ No newline at end of file
+   <MSYS inst dir>\mingw32\include and <MSYS inst dir>\mingw32\bin respectively.
+
+To compile file_sdk_sample:
+
+1. Downoad cxxopts: https://github.com/jarro2783/cxxopts/archive/master.zip .
```

Comment:
Isn't this just one hpp file? maybe just add it to the source?

Reply:
license problems?

---
In reply to: [139311327](https://github.com/AzureAD/rms-sdk-for-cpp/pull/153#discussion_r139311327) [](ancestors = 139311327)

Reply:
what is the license?
I don't think that someone will distribute the sample app commercially.